README.md

```diff
@@ -170,3 +163,7 @@ Request a Slack invitation via <concordbft@gmail.com>.
 
 concord-bft is available under the [Apache 2 license](LICENSE).
 
+## vmware Product & Research
```

Comment:
nit - VMware

Reply:
Nah, not a "nit". Actually very important:

I fetched it from the logo: https://www.vmware.com/

It seems logo and written are not consistent (hate it when it's that way...)